CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 ### Bug fixes
 
+* [#12](https://github.com/dduugg/yard-sorbet/pull/12): Fix parsing of recursive custom paremeterized types.
```

Comment:
Extremely minor nit: let's follow the rubocop [convention](https://github.com/rubocop-hq/rubocop/blob/master/CHANGELOG.md) and keep these in ascending chronological order